examples/fabric-admin/scripts/run_fabric_source_x86.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# Kill fabric-admin if it is running
```

Comment:
why don't we call `stop_fabric_source.sh` instead of having the same code inside?